scripts/update_specs_from_pricing.py

```diff
@@ -145,14 +115,13 @@ def get_mq_pricing():
         'mq.m5.2xl': 'mq.m5.2xlarge',
         'mq.m5.4xl': 'mq.m5.4xlarge'
     }
-
     results = {}
     for page in get_paginator('AmazonMQ'):
         for price_item in page.get('PriceList', []):
             products = json.loads(price_item)
             product = products.get('product', {})
             if product:
-                if product.get('productFamily') == 'Broker Instances':
+                if product.get('productFamily') in ['Broker Instances']:
                     if not results.get(region_map[product.get('attributes').get('location')]):
                         results[region_map[product.get('attributes').get('location')]] = set()
                     usage_type = product.get('attributes').get('usagetype').split(':')[1]
```

Comment:
[`instanceType` is not present, so `usagetype` must be parsed](https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonMQ/current/index.json)